### PR TITLE
Bumping up keep-core dependency to 1.3.0-pre

### PR DESCRIPTION
**Pull request by dimpar**
Bumping up keep-core dependency to 1.3.0-pre and hdwallet-provider to 1.0.37
Updating failed integration tests.

### CODE REVIEW

File: solidity/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
   },
   "homepage": "https://github.com/keep-network/keep-ecdsa",
   "dependencies": {
-    "@keep-network/keep-core": ">1.2.4-rc <1.2.4",
+    "@keep-network/keep-core": ">1.3.0-pre <1.3.0",
```
Comment (pdyraga) [inline solidity/package.json:35]: `>1.3.0-pre <1.3.0-rc` perhaps?

File: solidity/test/BondedECDSAKeepIntegrationTest.js [hunk 2]
```diff
@@ -254,6 +256,9 @@ contract("BondedECDSAKeepFactory", async (accounts) => {
       keepTokenGrant.address
     )
 
+    await TokenStaking.link("GrantStaking", (await GrantStaking.new()).address)
+    await TokenStaking.link("Locks", (await Locks.new()).address)
```
Comment (pdyraga) [inline solidity/test/BondedECDSAKeepIntegrationTest.js:260]: Let's keep them next to `MinimumStakeSchedule` linkage.

File: solidity/package.json (hunk 1 above)
Comment (dimpar) [reply solidity/package.json:35]: we don't want to include release candidates?

Comment (pdyraga) [reply solidity/package.json:35]: `-pre` should refer to `-pre`s. We should also update the version of this package on the top of the file.

Comment (dimpar) [reply solidity/package.json]: done

File: solidity/test/BondedECDSAKeepIntegrationTest.js (hunk 2 above)
Comment (dimpar) [reply solidity/test/BondedECDSAKeepIntegrationTest.js]: done

**Pull request merged**